### ISSUE TITLE
Sidebar on different view controllers

### ISSUE THREAD

**Issue by velci123** [John-Lluch/SWRevealViewController#656]
I have used `SWRevealViewController` to create side bar. As in this I have used my `HomeViewController` as front and `SidebarViewController` as right. My `SidebarViewController` contain a UITableView(Having different subject names). I want to open a new ViewController say `QuestionsViewController` on selection of any subject. Now my problem is I want to use `SWRevealViewController` again to have a sidebar in `QuestionViewController` but in `SWRevealViewController` identifier for front segue `sw_front` is a constant string so how can I change it or having any other way to have that sidebar in `QuestionViewController` also.

In Short : Can we have same sidebars in different ViewControllers using `SWRevelViewController`

**Comment by nvenkat999** [comment]
Hi @velci123 , did you find a solution for your question. I also want to use revealcontroller (sidebar) throughout my app. Please post the solution if you have found it.

**Comment by iDevelopper** [comment]
You have to use the SWRevealViewControllerSeguePushController segue with an identifier of your choice or programmatically pushFrontViewController

**Comment by mangireddy58** [comment]
Hi @iDevelopper  i tried with SWRevealViewControllerSeguePushController but not worked for me. do have any demo please provide.

Thanks.

**Comment by akshay1318a** [comment]
where are you getting error @mangireddy58 ???

**Comment by mangireddy58** [comment]
Hi @akshay1318a i am getting this error while using SWRevealViewControllerSeguePushController on different viewcontroller. "unexpectedly found nil while unwrapping an Optional value"

**Comment by iDevelopper** [comment]
Hi @mangireddy58 ,

There is a simple example:

[SWLayout.zip](https://github.com/John-Lluch/SWRevealViewController/files/1107992/SWLayout.zip)

**Comment by mangireddy58** [comment]
@iDevelopper thank you. its  working fine.
